src/Workspaces/Core/MSBuild.BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj

```diff
@@ -27,6 +27,8 @@
     <PackageReference Include="Microsoft.Build" VersionOverride="17.3.2" ExcludeAssets="Runtime" PrivateAssets="All" />
     <PackageReference Include="Microsoft.Build.Framework" VersionOverride="17.3.2" ExcludeAssets="Runtime" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" VersionOverride="17.3.2" ExcludeAssets="Runtime" PrivateAssets="All" />
+    <!-- fix of vulnerability in 6.0.0 coming via Microsoft.Build.Tasks.Core -->
+    <PackageReference Include="System.Security.Cryptography.Xml" VersionOverride="6.0.1" ExcludeAssets="Runtime" PrivateAssets="All" />
```

Comment:
So we are explicitly using 6.01 here instead of the 7.0.1 from Directories.Packages.props?

Reply:
Yeah, I guess we don't need to do that. Thanks.